src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/AbstractIntegrationTest.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.VisualStudio.Razor.IntegrationTests;
 /// <item><description><see cref="IDisposable.Dispose"/></description></item>
 /// </list>
 /// </remarks>
-[IdeSettings(MinVersion = VisualStudioVersion.VS2022, RootSuffix = "RoslynDev", MaxAttempts = 2)]
+[IdeSettings(MinVersion = VisualStudioVersion.VS2022, RootSuffix = "RoslynDev", MaxAttempts = 1)]
```

Comment:
Intentional? I think you've gone back and forwards on this, so just flagging in case you didn't mean to commit the change.

Reply:
Thank you, unintentional. I would really LIKE to turn the re-tries off, but don't have the numbers to justify that as safe. I turn the MaxAttempts number down when running locally so that failures are not hidden if I launch the integration tests before stepping away for lunch or something, which is how it sneaks back in.